project/c1s.mk

```diff
@@ -9,7 +9,10 @@ GLOBAL_DEFINES += \
 	SYSPARAM_ALLOW_WRITE=1 \
 	WITH_LIB_CONSOLE=1 \
 	INPUT_GPT_AS_PT=0 \
-	GPT_PART=0
+	GPT_PART=0 \
+	BOOTLOADER_FB_ADDRESS=0xf1000000 \
+	LCD_WIDTH=1080 \
+	LCD_HEIGHT=2400
```

Comment:
Wouldn't it be better to have these in target/board/rules.mk like this:
```
MEMBASE := 0x90000000
MEMSIZE := 0x00F00000
LCD_WIDTH := 1080
..
```

Reply:
Thanks Ivaylo!
Moving...

Reply:
Hey Ivaylo,
Just have a question - we would still need to add these to GLOBAL_DEFINES somewhere for the compiler to pick it up. So, we could do what is done above, or with your solution

`LCD_WIDTH=$(LCD_WIDTH)` <- in a global_defines

What do you think would be the cleaner solution?

Reply:
Resolved in the new commit series